src/components/Main/Team/Profile.tsx

```diff
@@ -9,9 +10,9 @@ interface Props {
 function Profile(props: Props) {
   return (
     <div>
-      <img src="#" alt={`Headshot of ${props.name}`} />
-      <p>{props.name}</p>
-      <p className="p2">{props.position}</p>
+      {props.img==="" ? <p></p> : <img className={isMobile ? "teamImgMob d-block mx-auto" : "teamImg d-block mx-auto"} src={props.img} alt={`Headshot of ${props.name}`} />}
+      <p className="text-peacock pt-2 text-center">{props.name}</p>
+      <p className="p2 text-center">{props.position==="PR" ? "Public Relations" : props.position}</p>
```

Comment:
instead of a conditional here, look into a enum and put it at the top of the component. you need to "translate" for every role, i.e. VP -> Vice President, Hackathon -> Director of Hackathon, etc.